### ISSUE TITLE
FEATURE - Include `account_id` column in models

### ISSUE THREAD

**Issue by wasifaleemdd** [fivetran/dbt_quickbooks#17]
**Are you a Fivetran customer?** 
Daydream

**Describe the solution you'd like**
Add `account_id` column to the following models:
* `quickbooks__profit_and_loss`
* `quickbooks__balance_sheet` 

**Please indicate the level of urgency and business impact of this request** 
This is currently blocking our customers from viewing aggregated model data

**Comment by fivetran-joemarkiewicz** [comment]
Thanks for opening this Issue @wasifaleemdd!

Our team will work on assigning this task internally and will communicate here when a member of our team is working to include the `account_id` field to the profit and loss and balance sheet models.

**Comment by fivetran-joemarkiewicz** [comment]
Hi @wasifaleemdd 👋  

I was able to open PR #20 which incorporates the `account_id` into the balance sheet and profit and loss models. You can follow the progress from there and we will let you know when it is merged and a new version is released.

**Comment by fivetran-kristin** [comment]
Just an update on progress here: 

The PR has been approved and will be merged into master when Joe returns next week.  

In parallel, we are working on these other requests:
https://github.com/fivetran/dbt_quickbooks/issues/18
https://github.com/fivetran/dbt_quickbooks/issues/19

When all of this work is done, we'll create a new release.

**Comment by fivetran-joemarkiewicz** [comment]
FYI the new release `v0.2.3` has been cut and should be live on the dbt hub tomorrow morning!

Closing this ticket as this update has been included in the latest release.
